### PR TITLE
ticdc: refine changefeed related documentations

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -71,6 +71,12 @@ case-sensitive = false
 # 详情请参考：https://docs.pingcap.com/zh/tidb/stable/ticdc-bidirectional-replication#ticdc-双向复制
 # bdr-mode = false
 
+# changefeed 发生内部错误和异常时自动重试的时间间隔，默认值为 30 分钟。
+# 若 changefeed 发生内部错误和异常，且持续时间超过该参数设置的时间间隔，changefeed 会进入 failed 状态。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:75]:
```suggestion
# 若 changefeed 发生内部错误和异常，且持续时间超过该参数设置的时间，changefeed 会进入 failed 状态。
```

File: ticdc/ticdc-ddl.md [hunk 2]
```diff
@@ -100,6 +100,10 @@ rules = ['test.t*']
 | `RENAME TABLE test.t1 TO ignore.t1, test.t2 TO test.t22;` | 报错 | 新的库名 ignore 不符合 filter 规则 |
 | `RENAME TABLE test.t1 TO test.t4, test.t3 TO test.t1, test.t4 TO test.t3;` | 报错 | 在一条 DDL 中交换 test.t1 和 test.t3 两个表的名字，TiCDC 无法正确处理。请参考错误提示提示信息处理。 |
 
+### 需要注意的 DDL 操作
+
+1. 当你在上游跨库执行形如 `create table db1.t1 like t2` 的 DDL 时，TiCDC 会尝试将该 DDL 同步到下游，但是由于 TiDB 传输给 DDL 信息中并不会给出 `t2` 所在的库名，因此 TiCDC 可能无法正确地执行该条 DDL 到下游。因此，建议在执行跨库的 DDL 时，显式的指定所有的库名，如 `create table db1.t1 like db2.t2` 。
```
Comment (qiancai) [inline ticdc/ticdc-ddl.md:105]:
```suggestion
当在上游执行跨数据库的 DDL 语句（如 `CREATE TABLE db1.t1 LIKE t2`）时，建议在 DDL 语句中显式地指定所有表的库名（如 `CREATE TABLE db1.t1 LIKE  db2.t2`）。否则，由于缺少库名信息，TiCDC 可能无法正确地执行该条 DDL 到下游。
```

File: ticdc/ticdc-ddl.md [hunk 3]
```diff
@@ -100,6 +100,10 @@ rules = ['test.t*']
 | `RENAME TABLE test.t1 TO ignore.t1, test.t2 TO test.t22;` | 报错 | 新的库名 ignore 不符合 filter 规则 |
 | `RENAME TABLE test.t1 TO test.t4, test.t3 TO test.t1, test.t4 TO test.t3;` | 报错 | 在一条 DDL 中交换 test.t1 和 test.t3 两个表的名字，TiCDC 无法正确处理。请参考错误提示提示信息处理。 |
 
+### DDL 语句注意事项
+
+当在上游执行跨数据库的 DDL 语句（如 `CREATE TABLE db1.t1 LIKE t2`）时，建议在 DDL 语句中显式地指定所有的库名（如 `CREATE TABLE db1.t1 LIKE  db2.t2`）。否则，由于缺少库名信息，TiCDC 可能无法正确地执行该条 DDL 到下游。
```
Comment (qiancai) [inline ticdc/ticdc-ddl.md:105]:
```suggestion
当在上游执行跨数据库的 DDL 语句（如 `CREATE TABLE db1.t1 LIKE t2`）时，建议在 DDL 语句中显式地指定所有的库名（如 `CREATE TABLE db1.t1 LIKE db2.t2`）。否则，由于缺少库名信息，TiCDC 可能无法正确地执行该条 DDL 到下游。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-ddl.md [hunk 4]
```diff
@@ -100,6 +100,10 @@ rules = ['test.t*']
 | `RENAME TABLE test.t1 TO ignore.t1, test.t2 TO test.t22;` | 报错 | 新的库名 ignore 不符合 filter 规则 |
 | `RENAME TABLE test.t1 TO test.t4, test.t3 TO test.t1, test.t4 TO test.t3;` | 报错 | 在一条 DDL 中交换 test.t1 和 test.t3 两个表的名字，TiCDC 无法正确处理。请参考错误提示提示信息处理。 |
 
+### DDL 语句注意事项
+
+当在上游执行跨数据库的 DDL 语句（如 `CREATE TABLE db1.t1 LIKE t2`）时，建议在 DDL 语句中显式地指定所有的库名（如 `CREATE TABLE db1.t1 LIKE db2.t2`）。否则，由于缺少库名信息，跨数据库的 DDL 语句可能无法正确地在下游执行。
```
Comment (qiancai) [inline ticdc/ticdc-ddl.md:105]:
```suggestion
当在上游执行跨数据库的 DDL 语句（如 `CREATE TABLE db1.t1 LIKE t2`）时，建议在 DDL 语句中显式地指定所有相关的库名（如 `CREATE TABLE db1.t1 LIKE db2.t2`）。否则，由于缺少库名信息，跨数据库的 DDL 语句可能无法正确地在下游执行。
```

**Review by qiancai — APPROVED** [review]

File: releases/release-7.5.0.md [hunk 5]
```diff
@@ -184,6 +184,7 @@ TiDB 7.5.0 为长期支持版本 (Long-Term Support Release, LTS)。
 | TiCDC | [`flush-worker-num`](/ticdc/ticdc-changefeed-config.md) | 新增 | 控制 redo 模块中上传文件 worker 的数量。 |
 | TiCDC | [`sink.column-selectors`](/ticdc/ticdc-changefeed-config.md) | 新增 | 控制 TiCDC 将增量数据分发到 Kafka 时，只发送指定的列的数据变更事件。 |
 | TiCDC | [`sql-mode`](/ticdc/ticdc-changefeed-config.md) | 新增 | 设置 TiCDC 解析 DDL 时使用的 SQL 模式，默认值和 TiDB 的默认 SQL 模式一致。 |
+| TiCDC | [`changefeed-error-stuck-duration`](/ticdc/ticdc-changefeed-config.md) | 新增 | 控制 changefeed 发生内部错误和异常时允许自动重试的时间。 |
```
Comment (qiancai) [inline releases/release-7.5.0.md:187]:
```suggestion
| TiCDC | [`changefeed-error-stuck-duration`](/ticdc/ticdc-changefeed-config.md) | 新增 | 控制 changefeed 发生内部错误或异常时允许自动重试的时间。 |
```

File: ticdc/ticdc-changefeed-config.md [hunk 6]
```diff
@@ -71,6 +71,12 @@ case-sensitive = false
 # 详情请参考：https://docs.pingcap.com/zh/tidb/stable/ticdc-bidirectional-replication#ticdc-双向复制
 # bdr-mode = false
 
+# changefeed 发生内部错误和异常时允许自动重试的时间，默认值为 30 分钟。
+# 若 changefeed 发生内部错误和异常，且持续时间超过该参数设置的时间，changefeed 会进入 failed 状态。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:75]:
```suggestion
# changefeed 发生内部错误或异常时允许自动重试的时间，默认值为 30 分钟。
# 若 changefeed 发生内部错误或异常，且持续时间超过该参数设置的时间，changefeed 会进入 failed 状态。
```

File: ticdc/ticdc-changefeed-config.md [hunk 7]
```diff
@@ -71,6 +71,12 @@ case-sensitive = false
 # 详情请参考：https://docs.pingcap.com/zh/tidb/stable/ticdc-bidirectional-replication#ticdc-双向复制
 # bdr-mode = false
 
+# changefeed 发生内部错误或异常时允许自动重试的时间，默认值为 30 分钟。
+# 若 changefeed 发生内部错误或异常，且持续时间超过该参数设置的时间，changefeed 会进入 failed 状态。
+# 当 changefeed 处于 failed 状态时，需要手动重启 changefeed 才能恢复。
+# 配置格式为 "h m s"，例如 "1h30m30s"
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:77]:
```suggestion
# 配置格式为 "h m s"，例如 "1h30m30s"。
```

File: ticdc/ticdc-changefeed-overview.md [hunk 8]
```diff
@@ -17,7 +17,7 @@ Changefeed 是 TiCDC 中的单个同步任务。Changefeed 将一个 TiDB 集群
 
 - Normal：同步任务正常进行，checkpoint-ts 正常推进。处于这个状态的 changefeed 会阻塞 GC 推进。
 - Stopped：同步任务停止，由于用户手动暂停 (pause) changefeed。处于这个状态的 changefeed 会阻挡 GC 推进。
-- Warning：同步任务报错，由于某些可恢复的错误导致同步无法继续进行。处于这个状态的 changefeed 会不断尝试继续推进，直到状态转为 Normal。最大重试时间为 30 分钟，超过该时间，changefeed 会进入 failed 状态。 处于这个状态的 changefeed 会阻挡 GC 推进。
+- Warning：同步任务报错，由于某些可恢复的错误导致同步无法继续进行。处于这个状态的 changefeed 会不断重试，试图继续推进，直到状态转为 Normal。默认重试时间为 30 分钟（可以通过 [`changefeed-error-stuck-duration`](/ticdc/ticdc-changefeed-config.md) 调整），超过该时间，changefeed 会进入 failed 状态。 处于这个状态的 changefeed 会阻挡 GC 推进。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-overview.md:20]:
```suggestion
- Warning：同步任务报错，由于某些可恢复的错误导致同步无法继续进行。处于这个状态的 changefeed 会不断重试，试图继续推进，直到状态转为 Normal。默认重试时间为 30 分钟（可以通过 [`changefeed-error-stuck-duration`](/ticdc/ticdc-changefeed-config.md) 调整），超过该时间，changefeed 会进入 Failed 状态。处于这个状态的 changefeed 会阻挡 GC 推进。
```

File: ticdc/ticdc-changefeed-config.md [hunk 9]
```diff
@@ -71,6 +71,12 @@ case-sensitive = false
 # 详情请参考：https://docs.pingcap.com/zh/tidb/stable/ticdc-bidirectional-replication#ticdc-双向复制
 # bdr-mode = false
 
+# changefeed 发生内部错误或异常时允许自动重试的时间，默认值为 30 分钟。
+# 若 changefeed 发生内部错误或异常，且持续时间超过该参数设置的时间，changefeed 会进入 failed 状态。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:75]:
```suggestion
# 若 changefeed 发生内部错误或异常，且持续时间超过该参数设置的时间，changefeed 会进入 Failed 状态。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**